docs/tutorial/image_seg.md

```diff
@@ -1,3 +1,168 @@
 # Training for Semantic Segmentation
 
-Semantic segmentation is in preparation.
+This guide trains a neural network model to semantic segmentation of [CamVid](http://mi.eng.cam.ac.uk/research/projects/VideoRec/CamVid/) dataset on GPU server.
+
+<img src="../_static/camvid.jpg" width="600">
+
+## Preparation
+
+The CamVid dataset is available from SegNet tutorial on [Github](https://github.com/alexgkendall/SegNet-Tutorial), but the dataset contain fixed absolute path of the data like [`/SegNet/CamVid/test/0001TP_008550.png`](https://github.com/alexgkendall/SegNet-Tutorial/blob/master/CamVid/test.txt). We  changed the absolute paths to relative paths. You can download from our mirror site.
+
+```
+$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/camvid.tgz
+$ tar xzf camvid
```

Comment:
`tar xzf camvid` -> `tar xzf camvid.tgz`

Reply:
@iizukak This is the `tar xzf camvid` where I mentioned to you.

Reply:
@lm-jira Thanks. Fixed.